layout/theme.liquid

```diff
@@ -67,7 +63,76 @@
         </button>
       </div>
     </div>
-    {% include 'mobile-nav' %}
```

Comment:
can you add HTML comments denoting the start/end to the old snippets? I think it would go a long way in helping ppl navigate this file.

Eg 

// begin mobile nav